RevenueCatUI/Helpers/Package+VariableDataProvider.swift

```diff
@@ -142,6 +166,46 @@ extension Package: VariableDataProvider {
 
 // MARK: - Private
 
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, *)
+private extension Package {
+
+    func priceEndsIn00Cents(_ priceString: String) -> Bool {
+        guard let formatter = self.storeProduct.priceFormatter?.copy() as? NumberFormatter else {
+            Logger.warning(Strings.no_price_format_priceFormatter_unavailable)
+            return false
+        }
+
+        guard let price = formatter.number(from: priceString)?.doubleValue else {
+            Logger.warning(Strings.no_price_format_priceString_incompatible)
+            return false
+        }
+
+        let roundedPrice = round(price * 100) / 100.0
+        return roundedPrice.truncatingRemainder(dividingBy: 1) == 0
+    }
+
+    func roundAndTruncatePrice(_ priceString: String) -> String {
```

Comment:
Maybe we could make the naming of this method more clear? 🤔 I _think_ truncate here means removing the `.00` but "truncate" can also have a different meaning like "round" where they almost contradict each other. 

Could we name this like `formatAsZeroDecimalPlaces`?

Reply:
I like it! Done!